cmd/login/oidc.go

```diff
@@ -108,17 +108,25 @@ func handleOIDC(ctx context.Context, out io.Writer, errOut io.Writer, i *install
 		authResult.clientID = clientID
 	}
 
-	apiServerURL := i.K8sApiURL
+	var apiServerURL string
+	{
+		if internalAPI {
+			apiServerURL = i.K8sInternalApiURL
+		} else {
+			apiServerURL = i.K8sApiURL
+		}
+	}
+
 	caData := []byte(i.CACert)
 	err = VerifyIDTokenWithKubernetesAPI(authResult.token, apiServerURL, caData)
 	if err != nil {
-		return authInfo{}, microerror.Mask(err)
+		fmt.Fprintf(errOut, "%s\n\n", color.YellowString("OIDC flow succeeded but token verification returned error %s.", err.Error()))
```

Comment:
```suggestion
		fmt.Fprintf(errOut, "%s\n", color.YellowString("OIDC flow succeeded but token verification returned error %s.", err.Error()))
```

Remove extra newline character.